src/calc_electron_temperature.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020, the Aether Development Team (see doc/dev_team.md for members)
+// Full license can be found in License.md
+
+#include "../include/aether.h"
+
+// --------------------------------------------------------------------------
+// Hack to just set the electon temperature to the neutral temperature
```

Comment:
If this is a hack, recommend changing comment to something like:
```suggestion
// TODO (#ISSUE NUMBER): this currently just sets the electron temperature to the neutral temperature
```

Reply:
This is my hack, which has been there since the start of the code.....  Agreed that it should have an issue, but there are a lot of things that should have issues which don't yet.  And we still have like 35 of them! :-)

Reply:
So, would you like to make the issue @aaronjridley or should @Ionospheres ?